packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue

```diff
@@ -418,6 +418,11 @@ const selectInputProps = computed(() => {
       outline: $outline-size solid var(--onyx-color-base-primary-200);
     }
 
+    &__wrapper:has(.onyx-mini-search) {
+      // Add scroll padding, so items are not hidden beneath the search input
+      scroll-padding-top: 40px;
```

Comment:
Can this value be calculated with CSS variables? Because the height changes based on the density, right?

Reply:
Right, we can just use `--option-height`